### PR TITLE
Bump version to 0.6.0

### PR DESCRIPTION
**Pull request by toshipp**
Signed-off-by: Toshikuni Fukaya <toshikuni-fukaya@cybozu.co.jp>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -195,7 +209,8 @@ Only cosmetic changes.
 
 This is the first release.
 
-[Unreleased]: https://github.com/topolvm/topolvm/compare/v0.5.3...HEAD
+[Unreleased]: https://github.com/topolvm/topolvm/compare/v6.0.0...HEAD
+[0.6.0]: https://github.com/topolvm/topolvm/compare/v0.5.3...v6.0.0
```
Comment (ymmt2005) [inline CHANGELOG.md:213]:
```suggestion
[Unreleased]: https://github.com/topolvm/topolvm/compare/v0.6.0...HEAD
[0.6.0]: https://github.com/topolvm/topolvm/compare/v0.5.3...v0.6.0
```

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (toshipp) [reply CHANGELOG.md:213]: Oops

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Review by llamerada-jp — APPROVED** [review]
LGTM

**Pull request merged**